asapdiscovery-data/asapdiscovery/data/tests/test_openeye_file_parsing.py

```diff
@@ -0,0 +1,63 @@
+import pytest
+from asapdiscovery.data.testing.test_resources import fetch_test_file
+from asapdiscovery.data.utils import (
+    exp_data_to_oe_mols,
+    load_exp_from_sdf,
+    oe_load_exp_from_file,
+)
+
+
+@pytest.fixture
+def sdf_file():
+    return fetch_test_file("Mpro_combined_labeled.sdf")
+
+
+@pytest.fixture
+def smi_file():
+    return fetch_test_file("Mpro_combined_labeled.smi")
+
+
+@pytest.fixture
+def file_dict():
+    return {
+        "sdf": fetch_test_file("Mpro_combined_labeled.sdf"),
+        "smi": fetch_test_file("Mpro_combined_labeled.smi"),
+    }
+
+
+def test_load_from_sdf(sdf_file):
+    mols = load_exp_from_sdf(str(sdf_file))
+    assert len(mols) == 576
+    # This is the current behaviour, which I dislike. Should be possible to have it pull an id from the sdf file
```

Comment:
Can't we pull try and pull the title from the parsed SDF with openeye otherwise just use `None` ?